### PR TITLE
Decorated all dates.

### LINKED ISSUE #12: Cycle Count Model
Add Cycle Count Model, Controller and Views

### LINKED ISSUE #23: Decorate Dates
Decorate created, updated and requested dates to mm/dd/yyyy.

### PR DESCRIPTION
**Pull request by LucasAnderson1984**
Cleaned up dates to a formatted mm/dd/yyyy.
Closes #23

Also, using this section to close Cycle Count Model issue.
Closes #12

### CODE REVIEW

File: app/views/cycle_counts/_cycle_count.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <tr>
   <td><%= cycle_count.location.description %></td>
-  <td><%= cycle_count.requested_date %></td>
+  <td><%= cycle_count.decorate.formatted_requested_date %></td>
```
Comment (josephbridgwaterrowe) [inline app/views/cycle_counts/_cycle_count.html.erb]: I'm curious, why didn't you decorate in the controller?

Same question, why you didn't decorate in the controller where ever you've decorated models.

File: spec/decorators/cycle_count_decorator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          requested_date: requested_date,
+          created_at: created_at,
+          updated_at: updated_at
+         ).decorate
+  end
+  let(:created_at) { Faker::Date.forward(30) }
+  let(:updated_at) { Faker::Date.forward(30) }
+  let(:requested_date) { Faker::Date.forward(30) }
+
+  describe '#reformatted_created_at' do
```
Comment (josephbridgwaterrowe) [inline spec/decorators/cycle_count_decorator_spec.rb]: `reformatted_` vs `formatted_`

File: spec/features/cycle_counts/user_creates_a_new_cycle_count_spec.rb [hunk 3]
```diff
@@ -18,7 +18,7 @@
 
       expect(page).to have_content(I18n.t('cycle_counts.create.success'))
       expect(page).to have_content(locations[0].description)
-      expect(page).to have_content(requested_date)
+      expect(page).to have_content(requested_date.strftime('%m/%d/%Y'))
```
Comment (josephbridgwaterrowe) [inline spec/features/cycle_counts/user_creates_a_new_cycle_count_spec.rb:21]: You've used decorate in the test expectations below, but not here and above?

Comment (LucasAnderson1984) [inline spec/features/cycle_counts/user_creates_a_new_cycle_count_spec.rb:21]: I don't understand.  The object isn't brought in to decorate.  I'm trying to make sure that given(:requested_date) { Faker::Date.forward(30) } gets changed to what the formatted_requested_date would be.  Unless I create another given like this? given(:formatted_requested_date) { requested_date.strftime('%m/%d/%Y') }

Comment (josephbridgwaterrowe) [inline spec/features/cycle_counts/user_creates_a_new_cycle_count_spec.rb:21]: Got it, I'm being a spaz. :smile:

**Comment by josephbridgwaterrowe** [comment]
@LucasAnderson1984 looks good, time for a rebase.

**Pull request merged**